lib/core/hooks.js

```diff
@@ -83,6 +83,9 @@ export async function buildHook (moduleContainer, options, logger) {
       }
     }
   }
+  if (options.enableTracing) {
+    options.clientConfig.tracesSampleRate = options.enableTracing.tracesSampleRate ? options.enableTracing.tracesSampleRate : 1
```

Comment:
I suppose it can be just:
```suggestion
    options.clientConfig.tracesSampleRate = options.enableTracing.tracesSampleRate || 1
```